containers/release-candidate/Dockerfile

```diff
@@ -42,6 +43,10 @@ RUN \
     winetricks && \
   dnf clean all
 
+# Install node dependencies:
+#   github-wikito-converter - creates html doc from github wiki repo
+RUN npm install -g github-wikito-converter
```

Comment:
Reviewing again as you requested. I don't know how much data npm install downloads under the covers, but if we go to the trouble of calling dnf clean all to clear everything that was downloaded and cached by dnf install, is there any equivalent command we want to append to this line to clear any npm cache?  Other than that, I believe you're good to squash and merge.

Reply:
There could be. However, I noticed using this `github-wikito-converter` it fails to convert all pages to HTML it just links to README files which we don't want. So I am going to remove this line from the Dockerfile and go back to using pandoc for both docx and html.